x-pack/plugins/enterprise_search/server/lib/enterprise_search_config_api.ts

```diff
@@ -135,3 +139,13 @@ export const callEnterpriseSearchConfigAPI = async ({
     clearTimeout(timeout);
   }
 };
+
+export const warnMismatchedVersions = (enterpriseSearchVersion: string, log: Logger) => {
```

Comment:
I was 50/50 about pulling this out to a separate helper const vs. keeping it inline in the main fn, but I figured there was enough going on in the main fn that it was a little nicer to pull this out to its own thing 🤔 If you feel strongly otherwise though I can change it, LMK